server.py

```diff
@@ -47,4 +66,4 @@ async def chat_endpoint(chat_input: ChatInput):
     )
     tokenizer = LlamaTokenizer.from_pretrained(args.model, use_fast=False)
 
-    uvicorn.run(app, host="0.0.0.0", port=8000)
+    uvicorn.run(app, host="0.0.0.0", port=8001)
```

Comment:
What would be the need for this port change? Since Uvicorn's default port is 8000, it would be better to keep it as is. If port change is necessary, lets make this configurable with CLI args with argparser

Reply:
Oh sorry, this is because I and Sarah was using the same instance so If both were running at the same time, there might has been a port-conflict so I changed to 8001, I forgot to change to 8000

Reply:
I have just changed to 8000